spec/unit/resource_helpers_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers
```

Comment:
Then it would probably be better if all these includes in the specs were moved into the initial describe block to keep from polluting the global object space. For example, the path_helpers_spec would look like:
```ruby
require 'spec_helper'

describe CernerSplunk::PathHelpers do
  include CernerSplunk::PathHelpers

  describe 'filename_from_url' do
    let(:expected_filename) { 'my_filename.tgz' }
    it 'should return the filename in a URL' do
      url = 'https://cool.website.internet/place/my_filename.tgz'
      expect(filename_from_url(url)).to eq(expected_filename)
    end
  end
end
```